src/taxonomy_router.rs

```diff
@@ -208,7 +201,7 @@ impl Handler for TaxonomyRouter {
         // Fetching and getting values.
         // We can't use a GET http method here because the Fetch() DOM api
         // doesn't allow bodies with GET and HEAD requests.
-        payload_api!(fetch_values, Vec<GetterSelector>, ["channels", "get"], Method::Put, binary);
+        payload_api!(fetch_values, Vec<GetterSelector>, ["channels", "get"], Method::Put, simple);
```

Comment:
It seems that we don't need the simple!() macro anymore and we can re-inline its content in payload_api!()